integration-libs/epd-visualization/components/visual-picking/visual-picking-tab/product-filter/visual-picking-product-filter.component.ts

```diff
@@ -0,0 +1,27 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { VisualPickingProductFilterService } from './visual-picking-product-filter.service';
+
+@Component({
+  selector: 'cx-visual-picking-product-filter',
```

Comment:
Angular components' selectors share one global namespace. Therefore it would be good to scope all components of your feature library by prefixing them with the feature name/abbreviation `cx-epd-...`, e.g. `cx-epd-visual-picking-product-filter`

This applies to all components in this library.

Reply:
I will do that.